suitcase.py

```diff
@@ -39,34 +40,30 @@ def export(headers, filename):
                 desc_group.create_dataset('event_times', event_times)
                 data_group = desc_group.create_group('data')
                 ts_group = desc_group.create_group('timestamps')
+                [fill_event(e) for e in events]
```

Comment:
@danielballan -- @tacaswell and I sorted out the mongo issue.  It was because you were calling fill_event on the same event multiple times, which makes dataportal look at the keys labeled 'external', it finds a numpy array and tries to use that numpy array to call `filestore.retrieve`, which, obviously, does not work.  Inserting this list comprehension here and removing the one at line 45-46 (as well as the dict casts) works.